File: app/src/main/java/com/numero/sojodia/contract/SettingsContract.java
package com.numero.sojodia.contract;

import com.numero.sojodia.presenter.IPresenter;
import com.numero.sojodia.view.IView;

public interface SettingsContract {

    interface View extends IView<Presenter> {

        void showBusDataVersion(String version);
    }

    interface Presenter extends IPresenter {
    }
}

File: app/src/main/kotlin/com/numero/sojodia/contract/SettingsContract.kt
package com.numero.sojodia.contract

import com.numero.sojodia.presenter.IPresenter
import com.numero.sojodia.view.IView

interface SettingsContract {

    interface View : IView<Presenter> {

        fun showBusDataVersion(version: String)
    }

    interface Presenter : IPresenter
}
